Cargo.toml

```diff
@@ -12,16 +12,18 @@ license = "MIT"
 repository = "https://github.com/maoertel/diqwest"
 
 [features]
-default = []
+default = ["reqwest/default"]
 blocking = ["reqwest/blocking"]
+rustls-tls = ["reqwest/rustls-tls"]
 
 [dependencies]
 async-trait = "0.1"
 digest_auth = "0.3"
-reqwest = "0.11"
+reqwest = { version = "0.11", default-features = false }
 
-[target.'cfg(blocking)'.dependencies]
-reqwest =  { version = "0.11", features = ["blocking"] }
+# is this really needed?
+# [target.'cfg(blocking)'.dependencies]
+# reqwest =  { version = "0.11", features = ["blocking"] }
```

Comment:
Back from vacation, read through it again: Makes sense, thx for highlighting and opening the PR. 👋🏾 
```suggestion
```